### ISSUE TITLE
Add an isTouching() method

### ISSUE THREAD

**Issue by davidnagli** [davidnagli/Scratch-JS#11]
Method should have two possible parameters:

isTouching(x,y);

or

isTouching(sprite);

**Comment by skistaddy** [comment]
Did we solve this?

**Comment by skistaddy** [comment]
I thought I remember you texting me about it.

**Comment by davidnagli** [comment]
We never added it... But someone on Stack Overflow wrote a pretty good solution which still needs to be modified and implemented with Scratch-JS

However, I think we should work on more core / essential features for now (mouse click and key press events, lists, costumes and more pen functionality), and focus on more time consuming and non essential features. 

I'm gonna create a new milestone for us with various task issues so we can keep track of everything.

**Comment by skistaddy** [comment]
K.

**Comment by bjskistad** [comment]
Bump

**Comment by skistaddy** [comment]
Whats the overflow link?

**Comment by davidnagli** [comment]
[http://stackoverflow.com/questions/40952985/how-can-i-check-if-two-partially-transparent-images-intersect-touch-each-other](http://stackoverflow.com/questions/40952985/how-can-i-check-if-two-partially-transparent-images-intersect-touch-each-other)

**Comment by skistaddy** [comment]
Yeah, I have no idea how to use that either.

**Comment by skistaddy** [comment]
http://stackoverflow.com/questions/26834244/how-do-i-test-if-two-img-are-touching-each-other

**Comment by davidnagli** [comment]
haha... It's hard but try your best to make it work or comment on his answer asking for clarification or whatever.

We really need this feature.

**Comment by davidnagli** [comment]
No, that link is for something else. Based on that answer we would only be able to check if they outlines of the image elements, not the actual non-transperent parts.
